packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -901,6 +901,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
 
     if (!this.state.focused) {
+      if (this.props.openOnFocus && !this.state.isOpen && !this.props.disabled) {
```

Comment:
To match the existing behavior, `openOnFocus` should default to true provided that `allowFreeform` is false. Having `openOnFocus` default to false is a breaking behavior change.